python/test/test_polars.py

```diff
@@ -610,23 +605,23 @@ def test_polars_threshold_rho():
 def test_replace_binary_path():
     import os
     pl = pytest.importorskip("polars")
-    expr = pl.len().dp.noise(scale=1.)
+    expr = dp.len(scale=1.)
```

Comment:
`pl.len().dp.noise()` comes up several times in the docs: Should those occurrences be cleaned up too?